Add framebuffer workaround for installer black-screen on iGPU/APU hosts (#488)
xcp-ng/xcp-ng-org: docs/troubleshooting/installation-upgrade.md

  * alternate kernel
  * safe mode
* Try to boot with the `iommu=0` xen parameter.

> * If the screen goes black and the installer hangs right after GRUB on a machine with only an integrated GPU (for example an AMD Ryzen APU), try the `vga=normal fb=false` kernel parameters to disable the framebuffer.

Reviewer: Is the "with only an integrated GPU" really significant?

Author: I am not, unfortunately, able to reproduce that on a discrete GPU.

Reviewer 2: Since we don't know the full extent of the affected hardware, maybe we could phrase it in a more generic way?

Reviewer 2: Ping @gounthar.

> * If the screen goes black and the installer hangs right after GRUB, try the `vga=normal fb=false` kernel parameters to disable the framebuffer. This was reported on a machine with integrated graphics only (an AMD Ryzen APU), but the full range of affected hardware is not known, so the parameters are worth trying whatever the graphics.

Reviewer 2: Trying a wording that's less linked to a specific report.

```suggestion
* If the screen goes black and the installer hangs right after GRUB, try the `vga=normal fb=false` kernel parameters to disable the framebuffer. This is known to be necessary on some machines with integrated graphics only (such as an AMD Ryzen APU), but the full range of affected hardware is not known, so the parameters are worth trying whatever the graphics.
```
